plugins/org.locationtech.udig.project.ui/src/org/locationtech/udig/project/ui/internal/ApplicationGISInternal.java

```diff
@@ -94,28 +93,28 @@ public static FeatureEditorLoader getFeatureEditorLoader( SimpleFeature feature
                 new StructuredSelection(feature));
     }
 
-    public static MapEditorPart getActiveEditor() {
+    public static MapPart getActiveEditor() {
```

Comment:
maybe its worth to 
* rename the method to getActiveMapPart()
* create another one with the old name that delegates to `getActiveMapPart()` - to be API compatible and mark it as deprecated (since SDK Users or integrators may use it)

```
@deprectated
public static MapPart getActiveEditor() {
   return getActiveMapPart();
}

public static MapPart getActiveMapPart() {
}
```